cmd/goal/clerk.go

```diff
@@ -867,7 +868,7 @@ var dryrunCmd = &cobra.Command{
 		block := bookkeeping.Block{}
 		block.BlockHeader.Round = basics.Round(round)
 		block.BlockHeader.TimeStamp = timeStamp
-		proto := config.Consensus[protocol.ConsensusCurrentVersion]
+		proto := config.Consensus[protocol.ConsensusFuture]
```

Comment:
do we want this in master

Reply:
Maybe we want to wait until the Foundation gives us the right spec name?